images/update_kcp_stable/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.6
+LABEL build-date= \
+      com.redhat.build-host= \
+      description="This image provides binaries and a script to update image tags on Gitlab repo" \
+      distribution-scope="public" \
+      io.k8s.description="This image provides binaries and a script to update image tags on Gitlab repo" \
+      io.k8s.display-name="upgrade gitlab" \
+      maintainer="Pipeline Service" \
+      name="upgrade-gitlab" \
+      release="0.1" \
+      summary="Provides the latest release of upgrade-gitlab image." \
+      url="" \
+      vcs-ref=  \
+      vcs-type="git" \
+      vendor="Pipeline Service" \
+      version="0.1"
+WORKDIR /
+RUN mkdir /workspace && chmod 777 /workspace && chown 65532:65532 /workspace
+ENV HOME /tmp/home
+RUN mkdir $HOME && chmod 777 $HOME && chown 65532:65532 $HOME
+RUN YQ_VERSION="v4.24.5" && \
+    curl --fail -sSL -o "/usr/local/bin/yq" "https://github.com/mikefarah/yq/releases/download/$YQ_VERSION/yq_linux_amd64" && \
+    chmod 755 "/usr/local/bin/yq"
+RUN JQ_VERSION="1.6" && \
+    curl -sSL -o "/usr/local/bin/jq" "https://github.com/stedolan/jq/releases/download/jq-$JQ_VERSION/jq-linux64" && \
+    chmod +755 "/usr/local/bin/jq"
```

Comment:
`+755` -> `755`. Also please check hadolint ci results, it seems to be unhappy about something.